src/Client/Grpc/GrpcDurableTaskClient.cs

```diff
@@ -84,9 +84,18 @@ public override async Task<string> ScheduleNewOrchestrationInstanceAsync(
             Version = orchestratorName.Version,
             InstanceId = options?.InstanceId ?? Guid.NewGuid().ToString("N"),
             Input = this.DataConverter.Serialize(input),
-            ParentTraceContext = new P.TraceContext() { TraceParent = Activity.Current?.Id, TraceState = Activity.Current?.TraceStateString },
         };
 
+        if (Activity.Current?.Id != null)
+        {
+            request.ParentTraceContext.TraceParent = Activity.Current?.Id;
```

Comment:
Is `request` guaranteed to have a valid `ParentTraceContext`?

Reply:
Can you clarify what you mean by "valid `ParentTraceContext`"?

Reply:
@bachuv The question was whether `request.ParentTraceContext` was guaranteed to be non-`null`.

Reply:
@bachuv - I believe @philliphoff is asking if `request.ParentTraceContext` may be `null`. In which case trying to access to `.TraceParent` will throw.

Reply:
Got it, I'll update the code to add a null check for `request.ParentTraceContext`.